georocket-server/src/main/java/io/georocket/GeoRocket.java

```diff
@@ -305,15 +307,36 @@ protected static JsonObject loadGeoRocketConfiguration() throws IOException, Dec
   }
 
   /**
-   * Look up all environment variables and take all variables with the prefix
-   * <code>georocket</code>. Parse the value and put it into the config. Replace existing key entries.
-   * @param conf The config
+   * Match every environment variable against the config keys from
+   * {{@link ConfigConstants#getConfigKeys()}} and save the found values using
+   * the config key in the config object. The method is equivalent to calling
+   * <code>overwriteWithEnvironmentVariables(conf, java.lang.System.getenv())</code>
+   * @param conf the config object
    */
   private static void overwriteWithEnvironmentVariables(JsonObject conf) {
-    System.getenv().forEach((key, val) -> {
-      if (key.startsWith("georocket")) {
+    overwriteWithEnvironmentVariables(conf, System.getenv());
+  }
+
+  /**
+   * Match every environment variable against the config keys from
+   * {{@link ConfigConstants#getConfigKeys()}} and save the found values using
+   * the config key in the config object.
+   * @param conf the config object
+   * @param env the map with the environment variables
+   */
+  static void overwriteWithEnvironmentVariables(JsonObject conf,
+    Map<String, String> env) {
+    Map<String, String> names = ConfigConstants.getConfigKeys()
```

Comment:
I need both the original property name and the generated environment variable name a few lines below. The map is used as optimization so i don't have to transform the original property name for every check again.